internal/controller/statusreport/statusreport_adapter.go

```diff
@@ -71,10 +71,6 @@ func NewAdapter(context context.Context, snapshot *applicationapiv1alpha1.Snapsh
 // The status is reported to git provider if it is a component snapshot
 // Or reported to git providers which trigger component snapshots included in group snapshot if it is a group snapshot
 func (a *Adapter) EnsureSnapshotTestStatusReportedToGitProvider() (controller.OperationResult, error) {
-	if gitops.IsSnapshotCreatedByPACPushEvent(a.snapshot) && !gitops.IsGroupSnapshot(a.snapshot) {
```

Comment:
We don't need to report status for `override` snapshot either, so we can update it to code like this
```
if gitops.IsOverrideSnapshot(a.snapshot) {
     return controller.ContinueProcessing()
}
```

Reply:
Or maybe like this to filter out all manual snapshot?
```
if !gitops.IsSnapshotCreatedByPACEvent(a.snapshot) && !gitops.IsGroupsnapshot(a.snapshot) {
     return controller.ContinueProcessing()
}
```

Reply:
Or define a function `IsNotMaunalSnaphost()` to include `!gitops.IsSnapshotCreatedByPACEvent(a.snapshot) && !gitops.IsGroupsnapshot(a.snapshot)` then you can use it in other places of this PR.

Reply:
I went with that first one just because I'm only using it in one place anyway. What do you think?